docs/src/main/sphinx/sql/create-catalog.md

```diff
@@ -0,0 +1,81 @@
+# CREATE CATALOG
+
+## Synopsis
+
+```text
+CREATE CATALOG
+catalog_name
+USING connector_name
+[ WITH ( property_name = expression [, ...] ) ]
+```
+
+## Description
+
+Create a new catalog using the specified connector.
+
+The optional `WITH` clause is used to set properties on the newly created
+catalog. Property names can be double quoted, which is required if they contain
+special characters, like `-`. Refer to the [connectors
+documentation](/connector) to learn about all available properties. All
+property values must be varchars (single quoted), including numbers and boolean
+values.
+
+The query fails in the following circumstances:
+* A required property is missing.
+* An invalid property is set, for example there is a typo in the property name,
+  or a property name from a different connector was used.
+* The value of the property is invalid, for example a numeric value is out of
+  range, or a string value doesn't match the required pattern.
+* The value references an environmental variable that is not set on the
+  coordinator node.
```

Comment:
does the variable not need to exist on worker nodes too? Or do workers get the secret value "inlined" into the config (hence no longer being secrets on the worker)?

Reply:
It does not need to exist on worker nodes. I assume workers receive already resolved values from the coordinator and store them in memory.

Reply:
Whats what I understand from @dain

Reply:
https://github.com/trinodb/trino/pull/19883